packages/api-cardano-db-hasura/src/example_queries/transactions/filteredAggregateDataWithinTransaction.graphql

```diff
@@ -1,16 +1,12 @@
 query filteredAggregatedDataWithinTransaction(
     $hash: Hash32Hex!
+    $outputsAddress: String!
 ){
     transactions(
         where: { hash: { _eq: $hash }}
             ) {
         hash
-        inputs_aggregate ( where: { value: { _gt: "3842014" }}) {
```

Comment:
Is `inputs_aggregate` obsolete?

Reply:
No, but the test/query scope is just to demonstrate the field can be filtered, and that's covered by `outputs_aggregate`